cli/session.go

```diff
@@ -261,7 +269,8 @@ func getTools(toComplete string) []string {
 	return tools
 }
 
-func getMetrics(toComplete string) []string {
+// TODO(oxisto): This could be an interesting use case for 1.18 Go generics
+func getMetrics(_ string) []string {
```

Comment:
```suggestion
func getMetrics() []string {
```

Reply:
This is intentional (future proof). The `toComplete` holds the typed input, we might want to include this in the request as a filter.